index.html

```diff
@@ -42,7 +41,56 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="examples/demo.css">
     <link rel="stylesheet" href="examples/themes.css">
-
+  <!-- Navbar start -->
+    <nav class="navbar">
+        <img id="logo-img" src="./images/Boots-ground-02.png" alt="publiclab-logo">
```

Comment:
Let's make this a tiny bit smaller -- can we do... half this size? Also, i don't see this image file, do you need to include it? https://github.com/publiclab/community-toolbox/tree/main/images

Reply:
Yes, I have set the size of the LOGO-IMG and also add an image file.
Thank you  @jywarren
![Logo](https://user-images.githubusercontent.com/51679200/105144824-d0343f00-5b23-11eb-903a-cd6a66b74e50.png)